### ISSUE TITLE
Get toast service directly from bridge library

### ISSUE THREAD

**Issue by Ullfis** [aurelia-ui-toolkits/aurelia-materialize-bridge#136]
Thank you for your work on this excellent bridge.

I am using Typescript and had a little problem with the toast service. The library 'aurelia-materialize-bridge/toast/toastService' (from http://aurelia-ui-toolkits.github.io/demo-materialize/#/samples/dialogs/) could not be found.

```javascript
import { MdToastService } from 'aurelia-materialize-bridge/toast/toastService';

.. not found compile failed
```

and

```javascript
import { MdToastService } from 'aurelia-materialize-bridge';

compiled ok but error on run
```

So I tried this:

aurelia-materialize-bridge/src/index.js

```javascript
export * from 'aurelia-materialize-bridge/toast/toastService';
```

And made className optional:

aurelia-materialize-bridge/src/toast/toastService.js

```javascript
..
  show(message, displayLength, className?) {
..
```

Then this work:
```javascript
import { MdToastService } from 'aurelia-materialize-bridge';

compiles ok run ok
```

Alternative is to add bridge/toast/toastService definition:

../aurelia-ui-toolkits/aurelia-materialize-bridge@master/aurelia-materialize-bridge-toast.d.ts
```javascript
declare module 'aurelia-materialize-bridge/toast/toastService' {
    export class MdToastService {
        show(message: any, displayLength: any, className?: any): any;
    }
}
```

**Comment by Thanood** [comment]
Thanks for mentioning, @Ullfis. I definitely need to use this with TypeScript more. :smile: 

To be honest, I didn't really have time to take a deeper look at the babel-dts-generator so I can't decide what would be the best solution of the two you mentioned.

But I think the first proposal should work. I will try that by tomorrow - not sure if I can do it today.

**Comment by Thanood** [comment]
This solution works but still throws an error:
```
Cannot find module 'aurelia-materialize-bridge/toast/toastService'."
```
It seems to be ignored, though.

**Comment by Thanood** [comment]
fixed in  https://github.com/aurelia-ui-toolkits/aurelia-materialize-bridge/commit/797d3b1d26aca26879c0a3beed4f9828b6bdec5e

**Comment by Thanood** [comment]
Now this comes back at me when running ```gulp doc-generate``` which is a dependency of ```gulp prepare-release```:

```
Cannot find module 'aurelia-materialize-bridge/toast/toastService'.
```

**Comment by Ullfis** [comment]
I played around with this today and concluded that the export..

```javascript
export * from './toast/toastService';

or

export * from 'aurelia-materialize-bridge/toast/toastService';
```

..in _index.js_ is not needed in the definition file. 

The code of MdToastService is included when _gulp build-index_ is run. The definition file is created by _gulp build-es6-temp_ based on the _gulp build-index_ result file. The result file is not used anymore.


How to remove the export?
- use remove tags

_src/index.js:_

```javascript
// build-index-remove start
export * from './toast/toastService';
// build-index-remove end
```

and make some changes in the _build/tasks/build.js_ file:

```javascript
..line 15..
var jsName = paths.packageName + '.js';

// RegExp remove tags during insert.transform in build-index gulp task
var startTag = '//\\s*build-index-remove start';
var endTag = '//\\s*build-index-remove end';
var removeRegExp = new RegExp(startTag + '[^]+?' + endTag, 'g');

gulp.task('build-index', function(){
  var importsToAdd = [];

  return gulp.src(paths.source)
    .pipe(through2.obj(function(file, enc, callback) {
      file.contents = new Buffer(tools.extractImports(file.contents.toString("utf8"), importsToAdd));
      this.push(file);
      return callback();
    }))
    .pipe(concat(jsName))
    .pipe(insert.transform(function(contents) {
      return tools.createImportBlock(importsToAdd) + contents.replace(removeRegExp, '');
    }))
    .pipe(gulp.dest(paths.output));
});
..
```

Remember the
```javascript
contents.replace(removeRegExp, '');

in

    .pipe(insert.transform(function(contents) {
      return tools.createImportBlock(importsToAdd) + contents.replace(removeRegExp, '');
    }))
```

**Comment by Thanood** [comment]
Thanks a ton for investigating this!
Just as a reminder: you can also issue a pull request with that content. :smile: 

I will apply that today when I have some more time.

**Comment by Kukks** [comment]
Thanks for the fix! :)

**Comment by Thanood** [comment]
merged in #140
